README.md

```diff
@@ -85,7 +85,7 @@ Examples can be found in [optuna/optuna-examples](https://github.com/optuna/optu
 
 ## Integrations
 
-[Integrations modules](https://optuna.readthedocs.io/en/stable/tutorial/10_key_features/003_efficient_optimization_algorithms.html#integration-modules-for-pruning), which allow pruning, or early stopping, of unpromising trials are available for the following libraries:
+[Integrations modules](https://optuna-integration.readthedocs.io/en/stable/index.html), which allow pruning, or early stopping, of unpromising trials are available for the following libraries:
```

Comment:
The link seems to be broken.

Reply:
We might also want to point to the documentations of unmigrated `optuna.integration` modules.

Reply:
This link will be valid after the release of optuna-integration. Would you feel uncomfortable posting a link that is broken, even temporarily? The reason for putting the link to the `stable` version instead of the `latest` is to be consistent with the way optuna documentation has been written so far.

Reply:
I see, I think that's okay. Thank you for explanation.